data/ui_scripts/server_list/serverlist.lua

```diff
@@ -77,14 +118,85 @@ function trimtext(text, font, height, maxwidth)
 	return text
 end
 
+function MakeHeaderText(textHolder, f5_arg1, column, color, menu)
+	local state = CoD.CreateState(f5_arg1, nil, f5_arg1 + 48, nil, CoD.AnchorTypes.Left)
+	state.font = CoD.TextSettings.TitleFontSmall.Font
+	state.top = -6
+	state.height = 14
+	state.alignment = nil
+	state.glow = LUI.GlowState.None
+	state.color = color
+
+	local actionButton = LUI.UIButton.new( state )
+	actionButton.alpha = 1
+	actionButton.width = 20
+	actionButton.root = menu
+	actionButton.id = "button_focus_" .. f5_arg1
+	actionButton.ui_sorting = column.dataindex
+	actionButton.requireFocusType = FocusType.MouseOver
+	actionButton:registerEventHandler( "button_action", function ( element, event )
+		for i = 0, #element.root.images, 1 do -- Hide all images and show only the images related to the selected type of sorting
+			if i == element.ui_sorting then
+				element.root.images[i]:show()
+			elseif element.root.images[i] ~= nil then
+				element.root.images[i]:hide()
+			end
+		end
+		Engine.SetDvarInt("ui_sorting", element.ui_sorting)
+		element.root:processEvent( {
+			name = "button_alt1",
+			dispatchChildren = true
+		} )
+		Engine.PlaySound( CoD.SFX.MenuAppear )
+	end)
+
+	local selectImage = LUI.UIImage.new({
```

Comment:
Just highlight the column text instead, or make the other columns a darker color

Reply:
I try to but i did not find a way to make it work. So i used an image for now.